openfl/interface/aggregator.py

```diff
@@ -34,6 +34,7 @@
 from openfl.utilities import click_types
 from openfl.utilities.path_check import is_directory_traversal
 from openfl.utilities.utils import getfqdn_env
+from openfl.utilities.attestation import attestation_utils as attestation_utils
```

Comment:
I think it would be cleaner to simply do `from openfl.utilities.attestation.attestation_utils import AttestationManager`

Reply:
there are some generic util methods which are part of attestation_utils and not AttestationManager like fetch_attestation_env_vars. Hence importing such